### PR TITLE
Default foreman cache changed from file to redis

### PR DESCRIPTION
**Pull request by rmynar**
Default Foreman cache was changed from file to redis. We have to adjust tests to reflect this change.

### CODE REVIEW

File: tests/foreman/installer/test_installer.py [hunk 1]
```diff
@@ -1468,7 +1468,7 @@ def test_foreman_rails_cache_store(sat_non_default_install):
 
     :steps:
         1. Install Satellite.
-        2. Verify that foreman-redis package is installed.
+        2. Verify that foreman-redis package is not installed.
```
Comment (Griffin-Sullivan) [inline tests/foreman/installer/test_installer.py:1471]: Could we change the comment below `Verify foreman-rails-cache-store option works` to `does not work`. Also can we change the docstring and test name to reflect this is testing non-default cache. like `test_foreman_non_default_rails_cache_store`

**Review by Griffin-Sullivan — COMMENTED** [review]
Just a quick couple of changes to make it obvious what the non-default test is. Everything else looks good

**Review by adarshdubey-star — APPROVED** [review]
Ack Code-wise pending comments from Griffin.

File: tests/foreman/installer/test_installer.py (hunk 1 above)
Comment (rmynar) [reply tests/foreman/installer/test_installer.py:1471]: Actually, the installer option (foreman-rails-cache-store) has to work. It should allow users to install with non default cache. The default cache is what has been changed from "file" to "redis". I slightly updated comments and docstring. I hope it's better now. Please check.

**Review by pondrejk — APPROVED** [review]

**Pull request merged**